iutil.py

```diff
@@ -1054,3 +1054,16 @@ def trySetfilecon(src, dest):
         errors.extend((src, dst, e.strerror))
     if errors:
         raise Error, errors
+
+def adjust_fraction(fraction):
+    """ Adjusts the fraction to the interval from 0.0 to 1.0.
+
+        Should be applied to a parameter of gtk.ProgressBar.set_fraction()
+        to ensure the correct value.
```

Comment:
I think the doc string should stay here. Yes the code is self explanation but on the other hand, you have here information why this is required.